database/connection.py

```diff
@@ -0,0 +1,36 @@
+import textwrap
+import pyodbc
+import sqlalchemy as db
+import urllib
+
+
+class SQLConnectionObject:
+   def __init__(self):
+       self.driver = '{ODBC Driver 18 for SQL Server}'
+       self.server_name = 'iparametricsserver'
+       self.database_name = 'iParametrics'
+       self.server = '{server}.database.windows.net, 1433'.format(server=self.server_name)
+       self.username = "teamipara"
+       self.password = "*****"
```

Comment:
Suggestion (non-blocking): from a security standpoint, maybe we can use secure storage or environment variables to store DB auth info. This is not necessary right now as we are still early in development but could be something to keep in mind of in the future